src/main/java/io/vertx/mqtt/messages/codes/MqttSubAckReasonCode.java

```diff
@@ -0,0 +1,48 @@
+package io.vertx.mqtt.messages.codes;
+
+import io.netty.handler.codec.mqtt.MqttQoS;
+import io.netty.handler.codec.mqtt.MqttVersion;
+
+public enum MqttSubAckReasonCode {
```

Comment:
why doesn't it extend `MqttReasonCode` like others ?

Reply:
Common handling wasn't needed in vertx-mqtt for this message. Now added that interface for uniformity.